packages/markdown-docx/src/CiceroMarkToOOXMLRules.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { sanitizeHtmlChars } = require('./CiceroMarkToOOXMLHelpers');
+
+/**
+ * Inserts text.
+ *
+ * @param {string}  value Enclosing value of the OOXML tag
+ * @returns {string} OOXML tag for the text
+ */
+const TEXT_RULE = (value) => {
+    return `<w:r><w:t xml:space="preserve">${sanitizeHtmlChars(value)}</w:t></w:r>`;
```

Comment:
@algomaster99 These tags are not used for conversion. So any space between them will act as '    <value>' instead of '<value>'. However, in `wrapAroundDefaultDocxTags` we do not check for such. So no omission of spaces there.

Reply:
> So any space between them will act as ' <value>' instead of '<value>'

Any problems with that? This worked fine in the Word add-in. I think the parser we use, `xml-js`, is also smart enough to ignore such spaces.

Reply:
@algomaster99
It included the spaces also in  values of nodes if it is done similar to add-in. So i dedcided to do it in later way. I also posted pn Slack regarding this. I will also reconfirm it.

Reply:
@algomaster99 
![Screenshot from 2021-06-11 15-01-12](https://user-images.githubusercontent.com/59891164/121665188-e6886a80-cac5-11eb-8031-52af09763a7a.png)
Changed the `EMPHASIS_RULE` and failed test

Reply:
By value, do you mean - `<w:value />` or `<w:t>value</w:t>`?

I read your post on slack and I thought it was causing some problems for you. But I don't think there is any issue as I just tried on my local with spaces and breaks.

If you think the size of the string (in bytes) is an issue, you can add a package to uglify an OOXML string after transformation is done.

Reply:
That is because you literally put a space before and after your value. I am just asking to format the rest of the nodes.

Reply:
Let's discuss this in the meeting.

Reply:
> By value, do you mean - `<w:value />` or `<w:t>value</w:t>`?

I mean `<w:t>value</w:t>

>I read your post on slack and I thought it was causing some problems for you. But I don't think there is any issue as I just tried on my local with spaces and breaks.

Interesting, as it gave me errors (shown above).

> If you think the size of the string (in bytes) is an issue, you can add a package to uglify an OOXML string after transformation is done.

We can have a discussion about this in the meeting.